src/viewer/overlay/BobsBuddyUserConfig.tsx

```diff
@@ -0,0 +1,167 @@
+import styled from "styled-components";
+import React from "react";
+import Cookies from "js-cookie";
+// import { VerticalList } from "../../config/components/OverlayAdmin/OverlayAdmin";
+
+const Wrapper = styled.div`
+	display: inline-block;
+	background: #23272a;
+	border-radius: 3px;
+	border: 1px solid #141617;
+	position: absolute;
+
+	left: 0;
+	top: 50%;
+
+	transform: translateY(-50%);
+	z-index: 1;
+	transform-origin: 0 0;
+	transition: padding 0.5s, width 0.5s;
+
+	overflow: hidden;
+`;
+
+export enum UserViewControl {
+	SeeBobsBuddy = "see_bobs_buddy",
+	ShowDuringCombat = "show_during_combat",
+	ShowDuringShopping = "show_during_shopping",
+}
+
+const ViewerLabel = styled.label`
+	font-size: 12px;
+	font-weight: bold;
+	color: white;
+	text-decoration: underline;
+	white-space: nowrap;
+`;
+
+const VerticalList = styled.ul`
+	padding-left: 0;
+	list-style-type: none;
+	margin: 0;
+	& > li:not(:first-child) {
+		margin-top: 17px;
+	}
+`;
+
+const SectionHeader = styled.label`
+	font-size: 15px;
+	font-weight: 250;
+	text-decoration: underline;
+	display: block;
+	border-width: 100%;
+	margin-bottom: 7px;
+	color: red;
+	white-space: nowrap;
+`;
+
+const CookiesDuration = 1;
```

Comment:
Should probably be more than 1 day? And add the comment here.

```suggestion
const COOKIE_DURATION = 365; // days
```